apps/src/templates/sectionProgress/ProgressBox.jsx

```diff
@@ -76,7 +76,7 @@ export default class ProgressBox extends Component {
     };
 
     return (
-      <div style={boxWithBorderStyle}>
+      <div style={boxWithBorderStyle} data-testid="progress-box">
```

Comment:
Since this `ProgressBox` can look like a lot of different things, but for the ProgressIcon component to render correctly I wanted to see that there existed a ProgressBox at certain points.  That is why I added in this `testid` here.